openllm-core/src/openllm_core/config/configuration_auto.py

```diff
@@ -42,6 +42,27 @@
   )
 )
 
+# NOTE: This is the entrypoint when adding new model config
+CONFIG_SUPPORTED_ARCHITECTURES_MAPPING = OrderedDict(
+  sorted(
+    [
+      ('FlanT5Config', ('T5ForConditionalGeneration',)),
+      ('BaichuanConfig', ('BaiChuanForCausalLM', 'BaichuanForCausalLM')),
+      ('ChatGLMConfig', ('ChatGLMModel',)),
+      ('FalconConfig', ('FalconForCausalLM',)),
+      ('GPTNeoXConfig', ('GPTNeoXConfig',)),
+      ('DollyV2Config', ('DollyV2Config',)),
```

Comment:
This is just the GPTNeoXForCausal architecture right?

Reply:
Ahhh, I made sure to double check that I had everything but I forgot to save 😓

Reply:
Good catch! Thanks!